### ISSUE TITLE
When page loads and on first click scroll to anchor behavior is abnormal, after second click smooth scroll works perfect.

### ISSUE THREAD

**Issue by TechWACO** [cferdinandi/smooth-scroll#427]
Hi the problems is like this:

After the page loads and on the first click the behavior is abnormal. It doesn't jump right to the anchor and it is not a smooth scroll, it behaves as the computers has not enough memory and during the scrolling it is jumping by small chunks till it reaches the desired anchor. Then, when scrolling up and click on the link for the second time, it works perfect. 

This error happens in Chrome Version 66.0.3359.139 OS X, Brave 0.22.669 OS X 64bit, Chrome Version 66.0.3359.181-1 Ubuntu and Brave 0.22.669 Ubuntu   

Works well on first click right after loading page in Safari Version 11.1 (13605.1.33.1.2) OS X 64bit and Firefox 60.0 64bit  Ubuntu, also seems to work well in mobile browsers for iOS as Brave 1.5.7, Safari and Chrome, haven't check on Android.

Please check the reduced test case here: https://www.dropbox.com/sh/xpogibqresely49/AAAkJOck9zAZxKiWDiw87PVka?dl=0 , click on the link "Productos"

Thanks

**Comment by cferdinandi** [comment]
Hi there, thanks again for this! I did some quick testing, and it looks like the large hero images are what's causing the jank. 

I reduced the width on all the images (currently around 5000px for most of them) down to 1600px, and then ran them through an image optimizer. The average size dropped from 2.5mb to 250kb. The jank went away.

I suspect the browser is still loading and painting the images during the initial animation. Once one has completed, it works fine. Similarly, manually scrolling to the bottom first and then using the smooth scroll link removes the jank, too.

In short: optimize those images!

**About your script usage:** 

You're loading all 4 versions of smooth scroll. You only need one. I'd recommend the minified and polyfilled version.

**Quick Side Note:**

Thanks again for the test case. For future issues, a reduced test case involves removing absolutely everything except the bare minimum needed to reproduce issue. That means, in this case, pulling out jQuery, the Bootstrap scripts, and any unnecessary styling.

**Comment by TechWACO** [comment]
Hi!

Thanks for the quick update on the issue. I updated to the new version, left just the minified version and optimized the images. 

Seems to work fine now. 

Noted on the test case.

Thanks.
